### ISSUE TITLE
Installing LiveCD / custom ISO on Vultr; Note about SSH

### ISSUE THREAD

**Issue by ArmouredHeart** [LukeSmithxyz/landchad#93]
---
author: a retard
---

### STORY TIME 

I followed the basic tutorial and (after much distro-hopping) now have my own website, however during my first attempt soystemD reared it's canaanite head and filled my Vultr virtual disk and memory up with garbage and rendered it inoperable. In the end I simply installed *Devuan ASCII* and then upgraded it to *[Devuan Beowulf](https://www.devuan.org/os/documentation/dev1fanboy/en/upgrade-to-beowulf)* (with openrc) using the Vultr ISO library.
During this installation I was utterly confounded with how exactly to unmount/remove the "LiveCD" from the cloud server and could not find an answer anywhere! (My search history consists of variations on "LiveCD on remote server" and "How to change boot order on cloud server".) Eventually Vultr support was able to figure out what I was trying to do and issued the following instructions;

1. Navigate to the "Products" page that lists your Vultr servers
2. Click on the server in question
3. Click on "Settings"
4. Click on "Custom ISO"
5. Click on the large blue button labeled "Detach ISO and Reboot"

From then on it was smooth sailing except for getting the ssh daemon to run. The expected command `$ rc-update add sshd default` returned "does not exist" errors, but `$ rc-update add ssh default` and then `rc-service ssh start` were accepted. It seems the ssh daemon goes by different names at random for each Linux distribution, even ones so similar as *Debian* and *Devuan* .

### TL;DR 

SystemD is gay and installing a LiveCD/LiveISO on a remote server seems simple yet either undocumented or difficult to find documentation for. Including a brief section on this subject in the basic tutorial would be *extremely* useful. 
Also, the ssh daemon goes by different names arbitrarily. Weird.
